test/docgen/view/__snapshots__/class.test.ts.snap

```diff
@@ -1,32 +1,87 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`csharp snapshot 1`] = `
-" AuthorizationToken <a name=\\"Amazon.CDK.AWS.ECR.AuthorizationToken\\" id=\\"amazoncdkawsecrauthorizationtoken\\"></a>
+Object {
+  "constants": Array [],
+  "displayName": "AuthorizationToken",
+  "docs": Object {
+    "see": "https://docs.aws.amazon.com/AmazonECR/latest/userguide/registry_auth.html",
+    "summary": "Authorization token to access private ECR repositories in the current environment via Docker CLI.",
+  },
+  "fqn": "Amazon.CDK.AWS.ECR.AuthorizationToken",
+  "id": "@aws-cdk/aws-ecr.AuthorizationToken",
+  "initializer": undefined,
+  "instanceMethods": Array [],
+  "interfaces": Array [],
+  "properties": Array [],
+  "staticMethods": Array [
+    Object {
+      "displayName": "GrantRead",
+      "docs": Object {
+        "summary": "Grant access to retrieve an authorization token.",
+      },
+      "fqn": "Amazon.CDK.AWS.ECR.AuthorizationToken.GrantRead",
+      "id": "@aws-cdk/aws-ecr.AuthorizationToken.grantRead",
+      "parameters": Array [
+        Object {
+          "default": undefined,
+          "displayName": "Grantee",
+          "docs": Object {},
+          "fqn": "Amazon.CDK.AWS.ECR.AuthorizationToken.parameter.Grantee",
+          "id": "@aws-cdk/aws-ecr.AuthorizationToken.parameter.grantee",
```

Comment:
I'm not sure of the implications here, but shouldn't this be:

```suggestion
          "id": "@aws-cdk/aws-ecr.AuthorizationToken.grandRead.parameter.grantee",
```

If this is being used to render links, if two methods have the same parameter, this might get confusing.